src/classes/issues-processor.ts

```diff
@@ -312,6 +353,151 @@ export class IssuesProcessor {
       }
     }
 
+    if (this.options.mlflow && !issue.isStale && !issue.milestone) {
+      const createMentions = (logins: string[]): string =>
+        logins.map(login => `@${login}`).join(' ');
+
+      const createMarkdownComment = (message: string): string =>
+        `<!-- ${message} -->`;
+
+      const isMaintainer = (login: string): boolean =>
+        this.maintainers.includes(login);
+
+      const TAGS = {
+        assignMaintainer: createMarkdownComment('assign-maintainer'),
+        reminderToMaintainers: createMarkdownComment('reminder-to-maintainers'),
+        reminderToIssueAuthor: createMarkdownComment('reminder-to-issue-author')
+      };
+
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const daysSinceIssueCreated = IssuesProcessor.getDaysSince(
+          issue.created_at
+        ).toFixed(2);
+        issueLogger.info(
+          `Days since this issue was created: ${daysSinceIssueCreated}`
+        );
+        issueLogger.info(
+          `Assignees on this issue: ${issue.assignees.map(({login}) => login)}`
+        );
+        const hasMaintainerAssignee = issue.assignees.some(user =>
+          isMaintainer(user.login)
+        );
+        const issueComments = await this.listIssueComments(
+          issue,
+          issue.created_at
+        );
+        const lastComment: IComment | undefined =
+          issueComments.length > 0
+            ? issueComments[issueComments.length - 1]
+            : undefined;
+
+        if (
+          !hasMaintainerAssignee &&
+          !IssuesProcessor._updatedSince(
+            issue.created_at,
+            this.options.daysBeforeAssigneeReminder
+          )
+        ) {
+          issueLogger.info('This issue has no assignees');
+          if (
+            !(lastComment && lastComment.body?.includes(TAGS.assignMaintainer))
+          ) {
+            const maintainersToMention = [
+              'BenWilson2',
+              'dbczumar',
+              'harupy',
+              'WeichenXu123'
+            ];
+            const mentions = createMentions(maintainersToMention);
+            await this.createComment(
+              issue,
+              `${TAGS.assignMaintainer}\n${mentions} Please assign a maintainer and start triaging this issue.`
+            );
+          }
+          return;
+        }
+
+        if (!lastComment) {
+          return;
+        }
```

Comment:
@harupy Consider the case where a community member creates an issue and a maintainer assigns it to themselves but never leaves a comment. Seems like `lastComment` will be empty, so stalebot will just ignore the issue. Is that accurate? If so, we should change this so that the stalebot leaves a comment reminding the maintainer to respond.

Reply:
Can we add a test for this too?

Reply:
Fixed and added a test :)